### ISSUE TITLE
Ignoring the absence of a new line between methods

### ISSUE THREAD

**Issue by blablatdinov** [wemake-services/wemake-python-styleguide#2721]
### What's wrong

I have a class with two methods. WPS ignore the absence of a new line between methods

```python
"""Example of code."""
from typing import Protocol


class ClassExample(Protocol):
    """Class Example docstring."""

    async def first_method(self) -> int:
        """First method docstring."""
    async def second_method(self) -> int:
        """Second method docstring."""
```

### How it should be

I want to take violation about "stack" methods.

### Flake8 version and plugins

{
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.8.5",
    "system": "Darwin"
  },
  "plugins": [
    {
      "plugin": "darglint",
      "version": "1.8.1"
    },
    {
      "plugin": "flake8-bandit",
      "version": "4.1.1"
    },
    {
      "plugin": "flake8-broken-line",
      "version": "1.0.0"
    },
    {
      "plugin": "flake8-bugbear",
      "version": "23.7.10"
    },
    {
      "plugin": "flake8-commas",
      "version": "2.1.0"
    },
    {
      "plugin": "flake8-comprehensions",
      "version": "3.14.0"
    },
    {
      "plugin": "flake8-debugger",
      "version": "4.1.2"
    },
    {
      "plugin": "flake8-docstrings",
      "version": "1.7.0"
    },
    {
      "plugin": "flake8-eradicate",
      "version": "1.5.0"
    },
    {
      "plugin": "flake8-isort",
      "version": "6.0.0"
    },
    {
      "plugin": "flake8-quotes",
      "version": "3.3.2"
    },
    {
      "plugin": "flake8-rst-docstrings",
      "version": "0.3.0"
    },
    {
      "plugin": "flake8-string-format",
      "version": "0.3.0"
    },
    {
      "plugin": "mccabe",
      "version": "0.7.0"
    },
    {
      "plugin": "pep8-naming",
      "version": "0.13.3"
    },
    {
      "plugin": "pycodestyle",
      "version": "2.11.0"
    },
    {
      "plugin": "pyflakes",
      "version": "3.1.0"
    },
    {
      "plugin": "wemake-python-styleguide",
      "version": "0.18.0"
    }
  ],
  "version": "6.1.0"
}

### pip information

pip 20.1.1 from /Users/almazilaletdinov/code/moment/venv/lib/python3.8/site-packages/pip (python 3.8)
astor==0.8.1
attrs==23.1.0
bandit==1.7.5
darglint==1.8.1
docutils==0.20.1
eradicate==2.3.0
flake8==6.1.0
flake8-bandit==4.1.1
flake8-broken-line==1.0.0
flake8-bugbear==23.7.10
flake8-commas==2.1.0
flake8-comprehensions==3.14.0
flake8-debugger==4.1.2
flake8-docstrings==1.7.0
flake8-eradicate==1.5.0
flake8-isort==6.0.0
flake8-quotes==3.3.2
flake8-rst-docstrings==0.3.0
flake8-string-format==0.3.0
gitdb==4.0.10
GitPython==3.1.32
isort==5.12.0
markdown-it-py==3.0.0
mccabe==0.7.0
mdurl==0.1.2
pbr==5.11.1
pep8-naming==0.13.3
pycodestyle==2.11.0
pydocstyle==6.3.0
pyflakes==3.1.0
Pygments==2.16.1
PyYAML==6.0.1
restructuredtext-lint==1.4.0
rich==13.5.2
smmap==5.0.0
snowballstemmer==2.2.0
stevedore==5.1.0
typing-extensions==4.7.1
wemake-python-styleguide==0.18.0

### OS information

ProductName:		macOS
ProductVersion:		13.2
BuildVersion:		22D49

**Comment by sobolevn** [comment]
`ruff` formats this now.
